### PR TITLE
feat: added lockscreen ui

### PR DESCRIPTION
**Pull request by irshadCypherock**

### CODE REVIEW

File: packages/cysync-core/src/pages/Lockscreen/ForgotPasswordDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  const [isLoading, setIsLoading] = React.useState(false);
+
+  const onConfirm = async () => {
+    setIsLoading(true);
+    await getResetCySyncMethod()();
+    setIsLoading(false);
+  };
+
+  return (
+    <DialogBox width={500} $isModal>
```
Comment (ghost) [inline packages/cysync-core/src/pages/Lockscreen/ForgotPasswordDialog.tsx:33]: Let's add a TODO here to update this screen from Figma

**Review by ghost — DISMISSED** [review]

File: packages/ui/src/components/molecules/Dialog/DialogBox.tsx [hunk 2]
```diff
@@ -21,6 +21,31 @@ export interface DialogBoxProps
...
+  z-index: 99;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  background-color: rgba(0, 0, 0, 0.3);
+  backdrop-filter: blur(2px);
+`;
+
```
Comment (ghost) [inline packages/ui/src/components/molecules/Dialog/DialogBox.tsx:48]: Can we consider using an HTML dialog element here?

File: apps/desktop/src/main/utils/autoUpdater.ts [hunk 3]
```diff
@@ -80,6 +81,19 @@ class AutoUpdater {
...
+  private parseUpdateInfo(info: ElectronUpdateInfo | UpdateInfo): UpdateInfo {
+    return {
+      ...info,
+      releaseNotes:
+        info.releaseNotes && Array.isArray(info.releaseNotes)
+          ? info.releaseNotes.map(note =>
+              typeof note === 'string' ? note : note.note ?? '',
+            )
+          : info.releaseNotes,
+    };
```
Comment (ghost) [inline apps/desktop/src/main/utils/autoUpdater.ts]: Refactor into multiple statements for readability

**Review by ghost — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/molecules/Dialog/DialogBox.tsx (hunk 2 above)
Comment (irshadCypherock) [reply packages/ui/src/components/molecules/Dialog/DialogBox.tsx:48]: Will consider it when building global dialog handler

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**